services/motion/localizer.go

```diff
@@ -90,5 +90,5 @@ func (m *movementSensorLocalizer) CurrentPosition(ctx context.Context) (*referen
 	}
 
 	pose := spatialmath.NewPose(spatialmath.GeoPointToPoint(gp, m.origin), o)
-	return referenceframe.NewPoseInFrame(m.Name().Name, spatialmath.Compose(pose, m.calibration)), nil
+	return referenceframe.NewPoseInFrame(m.Name().ShortName(), spatialmath.Compose(pose, m.calibration)), nil
```

Comment:
1. How do we know that all `referenceframe.NewPoseInFrame` are passed the `ShortName()` in the first parameter?
2. What is the consequence to this change if we are inconsistent?
3. Are we sure that the slam localizer's frame is `Word`? https://github.com/viamrobotics/rdk/pull/3395/files#diff-392336dc3cc22e5757a046be5626409190e789534a12719ee193dea241a59851R46

Reply:
bump

Reply:
1. The frame name should always be the shortname
2. We are/should not be inconsistent
3. Returning a PoseInFrame doesn't make a ton of sense outside the context of the frame system.  It definitely shouldnt be world here though.